jepsen/scalardb/src/scalardb/core.clj

```diff
@@ -19,13 +18,17 @@
            (com.google.inject Guice)
            (java.util Properties)))
 
-(def ^:const RETRIES 5)
+(def ^:const RETRIES 8)
 (def ^:const RETRIES_FOR_RECONNECTION 3)
 
 (def ^:const COORDINATOR "coordinator")
 (def ^:const STATE_TABLE "state")
 (def ^:const VERSION "tx_version")
 
+(defn exponential-backoff
+  [r]
+  (Thread/sleep (* 1000 (reduce * (repeat r 2)))))
```

Comment:
Not that it makes any difference, but `reduce` can also take an initial value, so could be rewritten here as

```
(reduce * 1000 (repeat r 2))
```

Reply:
Yes, it is better. Thanks.